crates/cli/src/datadog_utils.rs

```diff
@@ -113,14 +113,33 @@ fn make_request(
     }
 }
 
+fn perform_request(request_builder: RequestBuilder) -> Result<reqwest::blocking::Response> {
+    let mut server_response = None;
+    let mut retry_time = 1;
+    for _ in 0..5 {
+        match request_builder.try_clone().unwrap().send() {
+            Ok(r) => server_response = Some(r),
+            Err(_) => {
+                eprintln!("Error when querying the datadog server");
```

Comment:
1. This should only be shown if we run in debug mode.
2. We should show more information: what request has been made? what url?

Reply:
I was debating adding additional information, e.g. the error message, should we here? FYI this is if the request actually fails (internet issue or whatever), not if it's okay but we get a 404/500 etc (those errors are handled after this call)

Reply:
1. Not descriptive enough. Our errors should describe the problem and help the user (good developer tools help developers, think about the rust compiler vs. any C compiler)
2. The real error is reported in the main function if it fails and if we add more debugging, we should only add it if `configuration.use_debug` is `true`

Reply:
sounds good, and I agree. only one annoying point is that we pass `debug` down many function calls. It might be better to transform the request methods to be struct methods of our own `Request` type that wraps reqwest, and we can easily store `debug` here once instead of passing it down many function calls, but this isn't needed though